### DISCUSSION TITLE
[issue] Can not import in python 3.9.12 with aesara 2.8.2.

### DISCUSSION THREAD

**Discussion by panwanke** [aesara-devs/aesara#1162] [Q&A] [answered]
When I import aesara in python, I get this error. It seems to related with mkl library, but I have installed numpy+mkl. Should I install oneAPI?

---------------------------------------------------------------------------
NoSectionError                            Traceback (most recent call last)
File d:\Miniconda3\envs\hddm39\lib\site-packages\aesara\configparser.py:237, in AesaraConfigParser.fetch_val_for_key(self, key, delete_key)
    236 try:
--> 237     return self._aesara_cfg.get(section, option)
    238 except InterpolationError:

File d:\Miniconda3\envs\hddm39\lib\configparser.py:781, in RawConfigParser.get(self, section, option, raw, vars, fallback)
    780 try:
--> 781     d = self._unify_values(section, vars)
    782 except NoSectionError:

File d:\Miniconda3\envs\hddm39\lib\configparser.py:1152, in RawConfigParser._unify_values(self, section, vars)
   1151     if section != self.default_section:
-> 1152         raise NoSectionError(section) from None
...
   2459     ]
   2460     if windows_styled_libs:
   2461         selected_lib = sorted(windows_styled_libs, key=sort_key)[-1]

FileNotFoundError: [WinError 3] The system cannot find the specified path: 'C:/Program Files (x86)/Intel/oneAPI/mkl/latest/lib/intel64'

**Comment by panwanke** [comment, marked as answer]
This probelm may occur when using it with pymc3 or 2.

**Comment by lt-brs** [comment]
I've just raised a similar issue : https://github.com/aesara-devs/aesara/issues/1236#event-7526468125
Did you find out how to fix it or the source of it ?
